src/components/ConversationIcon.vue

```diff
@@ -59,12 +59,15 @@
 import Star from 'vue-material-design-icons/Star.vue'
 import VideoIcon from 'vue-material-design-icons/Video.vue'
 
+import { getCapabilities } from '@nextcloud/capabilities'
 import { generateOcsUrl } from '@nextcloud/router'
 
 import NcAvatar from '@nextcloud/vue/dist/Components/NcAvatar.js'
 
 import { CONVERSATION } from '../constants.js'
 
+const supportsAvatar = getCapabilities()?.spreed?.features?.includes('avatar')
```

Comment:
`spreed.features` is Array of strings, so we cannot look for available features by its name as a key

Reply:
Ops right